frodo-kem/README.md

```diff
@@ -0,0 +1,63 @@
+# [RustCrypto] Frodo-KEM
+
+[![Crate][crate-image]][crate-link]
+[![Docs][docs-image]][docs-link]
+![Apache2/MIT licensed][license-image]
+[![Downloads][downloads-image]][crate-link]
+![build](https://github.com/mikelodder7/frodoKem/actions/workflows/frodo-kem/badge.svg)
+
+A pure rust implementation of [FrodoKEM Learning with Errors Key Encapsulation](https://frodokem.org/files/FrodoKEM-specification-20210604.pdf).
+
```

Comment:
You reference the original paper. But the frodokem.org reference the proposal for the ISO standard. Not sure if there are any differences in the specifications.

Reply:
No they're the same.

Reply:
However there are some updates in the later annex that include a salt but none of the libraries I tested against have implemented that yet. I'm happy to add it now or in a follow up PR.

Reply:
Okay I added the both eFrodo and Frodo with the ISO test vectors.